packages/lib/gpc/test/gpc.spec.ts

```diff
@@ -33,16 +37,46 @@ describe("gpc library (Precompiled Artifacts) should work", async function () {
       pods: {
         somePodName: {
           entries: {
-            ticketID: { isRevealed: true }
+            ticketID: {
+              isRevealed: true,
+              ...(includeList ? { isMemberOf: "admissibleTickets" } : {})
+            }
           }
         }
-      }
+      },
+      ...(includeTuple
+        ? {
+            tuples: {
+              someTupleName: {
+                entries: ["somePodName.ticketID", "somePodName.ticketID"],
+                ...(includeList ? { isMemberOf: "admissibleTicketPairs" } : {})
+              }
+            }
+          }
+        : {})
     };
     const proofInputs: GPCProofInputs = {
       pods: {
         somePodName: pod2
       },
-      watermark: includeWatermark ? { type: "int", value: 1337n } : undefined
+      watermark: includeWatermark ? { type: "int", value: 1337n } : undefined,
+      ...(includeList
+        ? {
+            membershipLists: {
+              admissibleTickets: [
+                sampleEntries2.ticketID,
+                sampleEntries2.ticketID
```

Comment:
Use another ticket ID?  If you're going to have a list of size 2, it might as well be testing 2 cases.